client/k8s.go

```diff
@@ -418,3 +441,58 @@ func (k K8sClient) GetCstorVolumeRestores(pvName string) (*cstorv1.CStorRestoreL
 	return cStorRestoreList, nil
 
 }
+
+// GetPVs returns a list of PersistentVolumes
+func (k K8sClient) GetPVs() (*corev1.PersistentVolumeList, error) {
+	pvs, err := k.K8sCS.CoreV1().PersistentVolumes().List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		return nil, err
+	}
+	return pvs, nil
+}
+
+// GetJivaVolumes returns a list of jivavolumes
+func (k K8sClient) GetJivaVolumes() (*jiva.JivaVolumeList, error) {
+	jv := jiva.JivaVolumeList{}
+	// NOTE: The resource name must be plural and the API-group should be present for getting CRs
+	err := k.K8sCS.Discovery().RESTClient().Get().AbsPath("/apis/openebs.io/v1alpha1").
+		Resource("jivavolumes").Do(context.TODO()).Into(&jv)
+	if err != nil {
+		return nil, err
+	}
+	return &jv, nil
+}
+
+// GetPVbyName gets a list of PVs by the with name in vols in order
+func (k K8sClient) GetPVbyName(vols []string) (*corev1.PersistentVolumeList, error) {
```

Comment:
- https://github.com/openebs/openebsctl/blob/355e835b79541a7be39879333eddb1228e516cb6/client/k8s.go#L239
 We have the above method already. Maybe we can delete the above one replace with this method.

Reply:
GetPVbyName returns a PersistentVolumeList with one or more PVs. It's not fully same though but I do plan to merge all of this functionality into a single `func (k K8sClient) GetPV(vols []string) (*corev1.PersistentVolumeList, error)` which can take nil, 0 or more names of volumes and give out PVs in a PVList appropriately.

Reply:
We can use this method, incase we need to get only one PV, we can take the name and pass it as a list in this method, and at the point of retrieval we can get the first index if nothing is wrong.

Reply:
This is being used to get a list of volumes when the user supplies a list of volumes via args like this, `kubectl openebs get vol pvc-1234 pvc-12345 pvc-123456`, in this case the pvcList gets a list of only these three PVs in the same order of arguments.